indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3357,22 +3327,18 @@ public Map<PartitionIdType, SequenceOffsetType> apply(List<Either<Throwable, Map
                   log.info("Successfully set endOffsets for task[%s] and resumed it", setEndOffsetTaskIds.get(i));
                 } else {
                   String taskId = setEndOffsetTaskIds.get(i);
-                  killTask(
-                      taskId,
-                      "Task [%s] failed to respond to [set end offsets] in a timely manner, killing task",
-                      taskId
-                  );
+                  killTask(taskId, "Failed to set end offsets, killing task");
```

Comment:
do we not need to add the taskId to the message?

Reply:
I don't feel so as this message will always be used in the context of this task id (say while displaying reason for failure on the web console).

In fact, I think we should get rid of the task id from all such kill messages (in a separate PR) as it is redundant. Removing it will simplify the failure message.

(side benefit: significantly reduced cardinality if we emit the failure reason as a dimension atleast until we have error codes)

Reply:
How will it appear in the overlord logs though? If the task id is not available there, then we should add it.

Reply:
Makes sense. All primary task runner implementations (`HttpRemote`, `Remote`, `Kubernetes`) are doing the right thing:

```
log.info("Shutdown [%s] because: [%s]", taskId, reason);
```